### DISCUSSION TITLE
rst readme unsupported?

### DISCUSSION THREAD

**Discussion by tlambert03** [chanzuckerberg/napari-hub#136] [Ideas]
I forget if has been discussed somewhere, but it looks like RST readme's aren't currently supported on the hub:

https://staging.napari-hub.org/plugins/napari-dv

The metadata for that repo doesn't specify a `long_description_format` ... which, for PyPI, implies RST, and indeed it renders fine there: https://pypi.org/project/napari-dv/

is that something we can support?

**Comment by neuromusic** [comment]
we're not going to have RST support for the launch next week, though we might be able to add support at a later time.

I'd recommend adding a `.napari/DESCRIPTION.md` file that is more focused on the usage of the plugin and drops the content from the README that is redundant with the metadata that we're presenting (python versions, installation, license, etc)

**Comment by tlambert03** [comment]
I'll just convert to markdown... don't love having duplicated content in readme and napari/description
